.github/workflows/publish-rule-book.yaml

```diff
@@ -0,0 +1,114 @@
+  PO4A_VERSION: "0.71"
+  PO4A_GH_URL: https://github.com/mquinson/po4a/releases/download
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [en, pl]
+        include:
+          - language: en
+            version: Version
+            # warning: ""
+          - language: pl
+            version: Wersja
+            # warning: "To jest nieopublikowana wersja zbudowana na GitHubie z commita ${GITHUB_SHA::7}"
+
+    steps:
+      - name: Set up repository
+        uses: actions/checkout@v4
+
+      - name: Replace rule book version with a warning
+        run: |
+          VERSION=$(grep -P "^[\ ]*${{ matrix.version }} \d{1,}.\d{1,}$" main_${{ matrix.language }}.tex)
+          RELEASE_PAGE="${{ github.server_url }}/${{ github.repository }}/releases"
+          WARNING="This is an unreleased version built on GitHub from commit ${GITHUB_SHA::7}. To download the latest official release, please go to \\\href{${RELEASE_PAGE}}{GitHub release page} or \\\href{${{ env.BOARD_GAME_GEEK_RULE_BOOK }}}{BoardGameGeek files section}."
+          sed -i "s@$VERSION@$WARNING@g" main_${{ matrix.language }}.tex
+
+      - name: Generate translations for ${{ matrix.language }}
+        if: ${{ matrix.language != 'en' }}
+        run: |
+          sudo apt-get install -y po4a  # required as standalone po4a is missing some deps
+          wget --quiet ${{ env.PO4A_GH_URL }}/v${{ env.PO4A_VERSION }}/po4a-${{ env.PO4A_VERSION}}.tar.gz  # sadly the version from apt isn't working
+          export PO4A_DIR="po4a-${{ env.PO4A_VERSION }}"
+          tar -xf ${PO4A_DIR}.tar.gz
+          export PERL5LIB="${PO4A_DIR}/lib:${PERL5LIB}"
+          ./${PO4A_DIR}/po4a -v --no-update po4a.cfg
+
+      - name: Compile rule book
+        uses: xu-cheng/latex-action@v3
+        with:
+          extra_system_packages: "inkscape"
+          latexmk_shell_escape: true
+          root_file: main_${{ matrix.language }}.tex
+
+      - name: Move file
+        run: |
+          mkdir ${{ matrix.language }}
+          mv main_${{ matrix.language }}.pdf ${{ matrix.language }}
+
+      # - name: Publish main_${{ matrix.language }}.pdf in artifacts repository
+      #   uses: peaceiris/actions-gh-pages@v4
+      #   with:
+      #     deploy_key: ${{ secrets.SSH_DEPLOY_KEY_BUILD_ARTIFACTS }}
+      #     external_repository: qwrtln/Homm3BG-build-artifacts
+      #     publish_branch: main_${{ matrix.language }}
+      #     publish_dir: ./main_${{ matrix.language }}
+      #     force_orphan: true
+
+      # Printable build for now is available in English only
+
+      - uses: actions/setup-python@v5
+        if: ${{ matrix.language == 'en' }}
+        with:
+          python-version: "3.11"
+
+      - name: Add pages to select hyperlinks
+        if: ${{ matrix.language == 'en' }}
+        run: |
+          find sections -type f | xargs -n 1 sed -i "s@\\\hypertarget@\\\pagetarget@g"
```

Comment:
`find sections -type f -execdir sed -i "..." '{}' +`

Also if you replace " with ', you'll need less escaping of \

Reply:
Mind making a suggestion? I'm afraid to break it xD